pkg/handler-launcher-com/cmd/v1/cmd.proto

```diff
@@ -104,6 +104,10 @@ message InterfaceBindingMigration{
   string Method = 1;
 }
 
+message FeaturesToDisable{
```

Comment:
2882124d6a28ab4707d7ac9e91cc194a0609a42b Avoid reading Libvirt internal data -  It is little bit miss-leading, the commit actual provides a countermeasure for know limitations of available Libvirt versions. Please adjust the commit little bit to express this.

Reply:
Done